vanilla-framework/vanilla-framework-react
# Move all styling local to components

Note: this branch was created from #14, do not merge until it has been merged.

## Done

Moved styling of components locally as per suggestion by @Lukewh in #13 

## QA

- Pull code
- Review code changes
- Run `npm run start`
- Ensure no errors

Fixes #13
